collect_app/src/main/java/org/bcss/collect/naxa/submissions/PreviousSubmissionListActivity.java

```diff
@@ -298,34 +301,40 @@ public boolean isLoading() {
     }
 
     private void loadNextPage() {
-        ServiceGenerator.createCacheService(ApiInterface.class)
+        ServiceGenerator.createService(ApiInterface.class)
```

Comment:
we used to use http cache to store ! submission that were already downloaded
Are we removing that feature altogehter with this commit?

Reply:
@hereyubaraj like we discussed, I have made the changes